ncm-authconfig/src/main/pan/components/authconfig/sssd.pan

```diff
@@ -76,29 +83,30 @@ type authconfig_sssd_local = {
 };
 
 type authconfig_sssd_domain  = {
+    "reconnection_retries" ? long
     "ldap" ? authconfig_sssd_ldap
     "ipa" ? authconfig_sssd_ipa
     "simple" ? authconfig_sssd_simple
     "local" ? authconfig_sssd_local
     "access_provider" ? sssd_provider_string
     "id_provider" ? sssd_provider_string
-    "auth_provider" ? sssd_provider_string
-    "chpass_provider" ? sssd_provider_string
-    "debug_level" : long = 0x01F0
+    "auth_provider" ? sssd_pauth_rovider_string
+    "chpass_provider" ? sssd_auth_provider_string
+    "debug_level" ? long
     "sudo_provider" ? string
     "selinux_provider" ? string
     "subdomains_provider" ? string
     "autofs_provider" ? string
     "hostid_provider" ? string
-    "re_expression" : string = "(?P<name>[^@]+)@?(?P<domain>[^@]*$)"
+    "re_expression" ? string                            # "(?P<name>[^@]+)@?(?P<domain>[^@]*$)"
```

Comment:
Why the extra white-space?

Reply:
It's just for remember the syntax but i can remove it